site/content/support.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Where to Go for Support"
+draft: false
+weight: 1100
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-OOO"
+---
+
+## Support Policy
+
+NGINX Gateway Fabric adheres to the support policy detailed at [nginx.com/support](https://nginx.com/support).
+
+## Commercial Support
+
+For paid customers of NGINX Gateway Fabric, commercial support is available by opening a support case at the [MyF5](https://my.f5.com/) portal.
+
+Generate a Support Package
```

Comment:
We don't have any  creating support package in our backlog -- https://github.com/nginxinc/nginx-gateway-fabric/issues/623 . we either need to add it to our plans cc @mpstefan  or let's remove this paragraph.

Reply:
The commercial support altogether is unclear to me. From the Readme:
![image](https://github.com/nginxinc/nginx-gateway-fabric/assets/78599298/6247f245-40d5-4862-ae45-21347c0c7f60)

Reply:
That is a moment in time statement that is being changed.  And this is part of that change.

Reply:
Yes, remove mention of a Support Package.